<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nim::NimbleBuildStepWidget</class>
 <widget class="QWidget" name="Nim::NimbleBuildStepWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="argumentsLabel">
       <property name="text">
        <string>Arguments:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="argumentsLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="resetButton">
       <property name="toolTip">
        <string>Reset to Default</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
